pkg/node_reputation/node_reputation.go

```diff
@@ -0,0 +1,282 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package nodereputation
+
+import (
+	"bytes"
+	"database/sql"
+
+	// import of sqlite3 for side effects
+	_ "github.com/mattn/go-sqlite3"
+	proto "storj.io/storj/protos/node_reputation"
+)
+
+// auditSuccessRatio finds the ratio of audit success from the success and failure fields of a given reputaion row struct
+func (row nodeReputationRecord) auditSuccessRatio() float32 {
+	res := float32(0)
+	total := row.auditSuccess + row.auditFail
+
+	if total > 0 {
+		res = float32(row.auditSuccess) / float32(total)
+	}
+	return res
+}
+
+/*
+  naiveScore is naive formula for obtaining a repuataion score (scalar)
+  this method favors uptime, hence being multiplied by 100
+  and nullifys a score if there is a case of data modification
+*/
+func (row nodeReputationRecord) naiveScore() float32 {
```

Comment:
since i don't think we're prepared to use this function yet, i'd rather not maintain it as part of the codebase until we're ready. is that okay?